cleverhans/attacks.py

```diff
@@ -838,7 +865,7 @@ def __init__(self, model, sess=None, dtypestr='float32', **kwargs):
 
     super(SaliencyMapMethod, self).__init__(model, sess, dtypestr, **kwargs)
 
-    self.feedable_kwargs = {'y_target': self.tf_dtype}
+    self.feedable_kwargs = tuple(['y_target'])
```

Comment:
I think you can use `('y_target',)`  which is shorter than `tuple(['y_target'])`

Reply:
Done